packages/fluentui/docs-components/src/knobs/types.ts

```diff
@@ -40,6 +40,12 @@ export type KnobRangeKnobComponentProps = KnobComponentProps & {
   unit: string;
 };
 
+export type KnobNumberKnobComponentProps = KnobComponentProps & {
+  min: number;
+  max: number;
+  step: number;
```

Comment:
```suggestion
  min: string;
  max: string;
  step: string;
```

Just to have the same as in `KnobRangeKnobComponentProps`

Reply:
Done